src/script/cpp_api/s_client.cpp

```diff
@@ -189,6 +189,46 @@ bool ScriptApiClient::on_punchnode(v3s16 p, MapNode node)
 	return blocked;
 }
 
+bool ScriptApiClient::on_item_use(ItemStack item, PointedThing pointed)
+{
+	SCRIPTAPI_PRECHECKHEADER
+
+	// Get core.registered_on_item_use
+	lua_getglobal(L, "core");
+	lua_getfield(L, -1, "registered_on_item_use");
+
+	// Push data
```

Comment:
To avoid repeating code several times I proposed it in a MR: https://github.com/minetest/minetest/pull/5548/files#diff-fc5bd7f599797bce163af517e57a314c